packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/flutter_test_config.dart

```diff
@@ -1,27 +1,57 @@
 import 'dart:async';
 
+import 'package:alchemist/alchemist.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:golden_toolkit/golden_toolkit.dart';
 
 import 'helpers/goldens_file_comparator.dart';
 
-Future<void> testExecutable(FutureOr<void> Function() testMain) async =>
-    GoldenToolkit.runWithConfiguration(
-      () async {
-        WidgetsApp.debugAllowBannerOverride = false;
+/// Flag indicating if the tests are running in a CI environment
+/// Note: The environment variable may differ based on the platform.
+const bool _isRunningInCi = bool.fromEnvironment('CI') ||
+    bool.fromEnvironment('GITHUB_ACTIONS') ||
```

Comment:
Let's keep only the bool.fromEnvironment('CI'). The rest can be removed since they seems to be project specific.

Reply:
Done.